megamek/src/megamek/client/ui/swing/MovementDisplay.java

```diff
@@ -804,6 +804,7 @@ private void updateButtons() {
         }
         boolean isMech = (ce instanceof Mech);
         boolean isInfantry = (ce instanceof Infantry);
+        boolean isTank = (ce instanceof Tank);
```

Comment:
It is possible now to do ce.isVehicle() or ce.isSupportVehicle() or ce.isCombatVehicle, so no need to cache this I guess. I suppose isVehicle is the correct one here.

Reply:
Want me to change it now or do it in a separate PR where I can change all of the older code at once? Windchild292 mentioned something similar in another PR of mine about moveMode,

I will be doing a PR to change out all instances of code like `moveMode == EntityMovementMode.INF_LEG`

Reply:
If you want you can add it to that other todo.